src/Symfony/Component/HttpClient/CurlHttpClient.php

```diff
@@ -109,7 +113,7 @@ public function request(string $method, string $url, array $options = []): Respo
                 $options['headers']['range'] ?? null,
             ];
 
-            if ('GET' === $method && !$options['body'] && $expectedHeaders === $pushedHeaders) {
+            if ('GET' === $method && $expectedHeaders === $pushedHeaders && !$options['body']) {
```

Comment:
Could you also explain this?

Reply:
Ah, I was pretty sure that such a question would be raised.

I'm using PHPStorm and an amazing plugin: [PHP EA Extended](https://plugins.jetbrains.com/plugin/7622-php-inspections-ea-extended-).
It gives you plenty of tips on how to improve the code you're writing.

In this exemple, it tells you that the condition could be optimized if we change the order of the conditions. Apparently, the fastest `if` should be first (_less OPCODE_) to gain in performance.
It's obviously micro-optimizations.

See the attached screenshot of the plugin on the original code.
![Screenshot_20190409_083050](https://user-images.githubusercontent.com/252042/55777977-dd448300-5aa1-11e9-818d-f19c8c968e24.png)